salt/modules/zabbix.py

```diff
@@ -1058,6 +1058,97 @@ def host_update(hostid, **connection_args):
         return ret
 
 
+def hostinventory_get(hostids, **connection_args):
```

Comment:
I think `host_inventory_get` might be a better name here. I think it's easier to read.